arcana/cli/store.py

```diff
@@ -51,10 +51,10 @@ def add(nickname, type, location, varargs, cache, user, password):
 Gives a data store saved in the config file ('~/.arcana/stores.yml') a new
 nickname
 
-Arguments
----------
+Arguments\n
```

Comment:
I looked up the recommended way to format Click help and it is to use new paragraphs for each argument description and capitalise the variable name, https://click.palletsprojects.com/en/8.1.x/documentation/#documenting-arguments

Reply:
Otherwise the changes look good, so feel free to merge

Reply:
I am ok to reverse this, however, it is not very readable in the terminal.


<img width="624" alt="Screen Shot 2022-04-27 at 3 26 59 pm" src="https://user-images.githubusercontent.com/38680187/165447010-c081d11d-d52f-47f2-927c-3f0a85940089.png">